onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java

```diff
@@ -61,11 +87,31 @@ public void onFailure(@NonNull Exception exception) {
                     @Override
                     public void onSuccess(UploadTask.TaskSnapshot taskSnapshot) {
                         Log.d(TAG, logFileName + " uploaded successful");
+                        String userResponse = taskSnapshot.getMetadata().getCustomMetadata("Response");
```

Comment:
Could you please put these strings in `donottranslate.xml`?

Reply:
@barbeau Thank you for reviewing the PR. I have added the strings to `donottranslate.xml`.